ext/curl/interface.c

```diff
@@ -3257,6 +3260,11 @@ PHP_FUNCTION(curl_getinfo)
 		if (ch->header.str) {
 			CAASTR("request_header", ch->header.str);
 		}
+#if LIBCURL_VERSION_NUM >= 0x074800 /* Available since 7.72.0 */
+		if (curl_easy_getinfo(ch->cp, CURLINFO_EFFECTIVE_METHOD, &s_code) == CURLE_OK) {
+			CAAS("method", s_code);
```

Comment:
```suggestion
			CAAS("effective_method", s_code);
```
Possibly?

Reply:
Thank you for the review @nikic. I thought to use `method` because `EFFECTIVE_URL` is currently set to `url`. But I think your suggestion makes more sense; let's gowith `effective_method`.

I will force-push with this change.